app/views/supported_permissions/index.html.erb

```diff
@@ -26,6 +26,20 @@
   } %>
 </div>
 
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/details", {
+      title: "What is a 'delegated' permission?"
+    } do %>
+      <p class="govuk-body">
+        When a permission is delegated, publishing managers are able to grant the
```

Comment:
This is a helpful explainer, but I'm not sure I understand the distinction between "delegated" and "delegatable" here. Isn't "delegatable" the right word here for someone being able to delegate a thing? Using the past participle feels like a thing that's already happened, rather than a thing that can happen

Reply:
I can see how this could be confusing, and I think the choice of distinguishing verbs isn't the most natural, but 'delegating' and 'granting' permissions are two separate concepts. We grant permissions to individuals, while we delegate the authority to grant permissions to publishing managers. I wonder if we need to clarify that here (and maybe also in the access and permissions documentation)?

Reply:
tl;dr 'delegated' == 'grantable by publishing managers'